pyzeebe/client/client.py

```diff
@@ -17,9 +16,9 @@ def __init__(self, hostname: str = None, port: int = None, credentials: BaseCred
             port (int): Port of the zeebe
             max_connection_retries (int): Amount of connection retries before client gives up on connecting to zeebe. To setup with infinite retries use -1
         """
-
-        self.zeebe_adapter = ZeebeAdapter(hostname=hostname, port=port, credentials=credentials,
-                                          secure_connection=secure_connection,
+        options = merge_options(hostname, port, credentials, secure_connection)
+        self.zeebe_adapter = ZeebeAdapter(hostname=options.hostname, port=options.port, credentials=options.credentials,
+                                          secure_connection=options.secure_connection,
```

Comment:
`Options` could have a `as_dict` method as well, so we could do `ZeebeAdapter(**options.as_dict(), ...)